src/Sql/Sql/Database/Model/AzureSqlDatabaseModel.cs

```diff
@@ -314,10 +320,10 @@ public AzureSqlDatabaseModel(string resourceGroup, string serverName, Management
             AutoPauseDelayInMinutes = database.AutoPauseDelay;
             MinimumCapacity = database.MinCapacity;
             HighAvailabilityReplicaCount = database.HighAvailabilityReplicaCount;
-            CurrentBackupStorageRedundancy = database.CurrentBackupStorageRedundancy;
-            RequestedBackupStorageRedundancy = database.RequestedBackupStorageRedundancy;
+            BackupStorageRedundancy = MapInternalBackupStorageRedundancyToExternal(database.CurrentBackupStorageRedundancy);
```

Comment:
This looks like the old ps; we want current/requested backup storage redundancy for sdk versions 1.50+

Reply:
fixed